### PR TITLE
Remodels update 1.6 -> master

### PR DESCRIPTION
**Pull request by Willster419**

### CODE REVIEW

**Comment by Arkhorse** [comment]
#38

File: modInfo.xml [hunk 1]
```diff
@@ -43359,12 +43332,12 @@
...
                       <visible>True</visible>
                       <packageName>Custom_Tank_Skins_Germany_Heavy_Maus_HierKommtDieMaus</packageName>
-                      <type>single1</type>
+                      <type>single_dropdown1</type>
                       <version>1.1.0</version>
                       <timestamp>131870937148056910</timestamp>
                       <size>3573608</size>
                       <description>"Die Sendung mit der Maus"</description>
-                      <devURL>https://wgmods.net/1882/details</devURL>
+                      <devURL>https://www.facebook.com/TankRaceSkins/\r\nhttps://wot.modhoster.de/mods/yodaschlumpf-race-tank#images\r\nhttps://www.modhoster.de/community/user/yodaschlumpf#t4\r\nhttps://wgmods.net/1882/details</devURL>
```
Comment (Willster419) [inline modInfo.xml]: this URL needs to be removed

File: modInfo.xml [hunk 2]
```diff
@@ -44081,8 +44063,8 @@
                       <name>Leopard 1 "Schneeleopard"</name>
                       <zipFile>Custom_Tank_Skins_G89_Leopard1_Mirukii_1.5.0.4_2019-06-03.zip</zipFile>
```
Comment (Willster419) [inline modInfo.xml]: this is the wrong zip file

File: modInfo.xml [hunk 3]
```diff
@@ -45844,64 +45775,6 @@
               <packageName>Custom_Tank_Skins_USA_Heavy</packageName>
               <type>multi</type>
               <packages>
-                <package>
```
Comment (Willster419) [inline modInfo.xml]: NOTE, not todo: these were moved to medium, it's not a heavy tank

File: modInfo.xml [hunk 4]
```diff
@@ -46137,15 +46014,26 @@
...
                       </medias>
                     </package>
+                    <package>
+                      <name>T34 B by HG_Skins</name>
+                      <zipFile>Custom_Tank_Skins_A13_T34_B_By_HG_Skins_1.4.1.2_2019-08-16.zip</zipFile>
+                      <crc>f</crc>
+                      <enabled>True</enabled>
+                      <visible>True</visible>
+                      <packageName>Custom_Tank_Skins_USA_Heavy_T34_B_Heavy_HG_Skins</packageName>
+                      <type>multi</type>
```
Comment (Willster419) [inline modInfo.xml]: this needs to be changed to `single1` type

**Pull request merged**